iocore/net/P_SSLUtils.h

```diff
@@ -34,6 +34,8 @@
 #include "records/I_RecCore.h"
 #include "P_SSLCertLookup.h"
 
+#define TS_OPENSSL_1_1_1_API (OPENSSL_VERSION_NUMBER >= 0x10101000L)
```

Comment:
I'm not sure if I like this. First, the name is confusing, in that it sounds it's specific to v1.1.1. But worse, this paints us in a corner of only supporting OpenSSL. At a minimum, this ought to move out to some autoconf check, ideally looking for a feature, and rename it accordingly.

Reply:
Done.